### PR TITLE
Add Pop!_OS instructions

### LINKED ISSUE #264: Add instructions for Pop!_OS
Pop!_OS is a GNOME-based distro built on Ubuntu, shipped by default on System76 computers. However, it does not use GNOME Software. The installation instructions would be similar to the Ubuntu ones, but not quite the same. Adding instructions for Pop!_OS users would help them get started since it may not be obvious which parts of the Ubuntu instructions to follow, and which to omit.

### PR DESCRIPTION
**Pull request by cassidyjames**
Fixes #264. Will probably artificially conflict with #265 but that will be any easy fix to resolve.

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -254,3 +254,34 @@
...
+  info: >  
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>For Pop!_OS OS 17.10, first add the official Flatpak PPA by running the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> sudo apt install software-properties-common --no-install-recommends
+          <span class="unselectable">$</span> sudo add-apt-repository ppa:alexlarsson/flatpak
+        </code></pre>
+        <p>For all versions of Pop!_OS, install flatpak from a terminal:</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: flatpak -> Flatpak

File: data/distro.yml [hunk 2]
```diff
@@ -254,3 +254,34 @@
...
       </li>
     </ol>
+
+- name: "Pop!_OS"
+  logo: "pop-os.svg"
+  info: >  
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>For Pop!_OS OS 17.10, first add the official Flatpak PPA by running the following in a terminal:</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: What about Pop!_OS 18.04 and later?

File: data/distro.yml [hunk 3]
```diff
@@ -254,3 +254,34 @@
...
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+        <p>Note: graphical installation of Flatpak apps may not be possible with elementary OS.</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: elementary OS?

File: data/distro.yml (hunk 1 above)
Comment (AsciiWolf) [reply data/distro.yml]: a terminal -> the Terminal

**Comment by cassidyjames** [comment]
@AsciiWolf, the reason for "the Terminal" in the elementary OS instructions is for consistency with how the terminal app is referred to in official elementary OS documentation (and therefore what users would be useful). In the Pop!_OS case, I'm mirroring the instructions from Ubuntu that were already here. I've also simplified the instructions here a bit due to https://github.com/flatpak/flatpak.github.io/issues/260#issuecomment-388057403

**Pull request merged**